api/api/spec/spec.yaml

```diff
@@ -8590,6 +8590,7 @@ paths:
     get:
       tags:
         - Agents
+      deprecated: true
       summary: "List agents without group"
       description: "Return a list with all the available agents without an assigned group"
       operationId: api.controllers.agent_controller.get_agent_no_group
```

Comment:
We should add the [deprecate_endpoint](https://github.com/wazuh/wazuh/blob/dccbb41b0dd74c7ac8d13a9158cd489600dd77cb/api/api/util.py#L427) decorator in the `api.controllers.agent_controller.get_agent_no_group` function to add the corresponding headers in the HTTP response.